### PR TITLE
nodeutilization: usage2KeysAndValues for constructing a key:value list for InfoS printing resource usage

### PR DESCRIPTION
**Pull request by ingvagabund**
Code deduplication

### CODE REVIEW

File: pkg/framework/plugins/nodeutilization/nodeutilization.go [hunk 1]
```diff
@@ -226,6 +226,26 @@ func classifyNodes(
 	return lowNodes, highNodes
 }
 
+func usage2KeysAndValues(usage map[v1.ResourceName]*resource.Quantity) []interface{} {
```
Comment (atiratree) [inline pkg/framework/plugins/nodeutilization/nodeutilization.go:229]:
```suggestion
func usageToKeysAndValues(usage map[v1.ResourceName]*resource.Quantity) []interface{} {
```

better to be consistent and not use number shortcuts in the names

File: pkg/framework/plugins/nodeutilization/nodeutilization.go [hunk 2]
```diff
@@ -351,24 +356,8 @@ func evictPods(
...
-					keysAndValues = append(keysAndValues, "Pods", quantity.Value())
-				}
-				for name := range totalAvailableUsage {
-					if !nodeutil.IsBasicResource(name) {
-						keysAndValues = append(keysAndValues, string(name), totalAvailableUsage[name].Value())
-					}
-				}
-
+				keysAndValues := usage2KeysAndValues(nodeInfo.usage)
+				keysAndValues = append(keysAndValues, "node", nodeInfo.node.Name)
```
Comment (atiratree) [inline pkg/framework/plugins/nodeutilization/nodeutilization.go:360]: might be better to keep the same order

File: pkg/framework/plugins/nodeutilization/nodeutilization.go [hunk 3]
```diff
@@ -351,24 +356,8 @@ func evictPods(
...
-				if quantity, exists := nodeInfo.usage[v1.ResourceCPU]; exists {
-					keysAndValues = append(keysAndValues, "CPU", quantity.MilliValue())
-				}
-				if quantity, exists := nodeInfo.usage[v1.ResourceMemory]; exists {
-					keysAndValues = append(keysAndValues, "Mem", quantity.Value())
-				}
-				if quantity, exists := nodeInfo.usage[v1.ResourcePods]; exists {
-					keysAndValues = append(keysAndValues, "Pods", quantity.Value())
-				}
-				for name := range totalAvailableUsage {
```
Comment (atiratree) [inline pkg/framework/plugins/nodeutilization/nodeutilization.go:366]: so the mixing of node and total stats was incorrect

Comment (ingvagabund) [reply pkg/framework/plugins/nodeutilization/nodeutilization.go:366]: Yeah. I probably misunderstood your previous comment.

File: pkg/framework/plugins/nodeutilization/nodeutilization.go (hunk 1 above)
Comment (ingvagabund) [reply pkg/framework/plugins/nodeutilization/nodeutilization.go]: Done

**Comment by atiratree** [comment]
lgtm

**Pull request merged**